docs/snap-clone.md

```diff
@@ -159,41 +162,50 @@ kubectl create -f snapshot.yaml
 
 ```console
 $ kubectl get volumesnapshot
-NAME               AGE
-rbd-pvc-snapshot   6s
+NAME               READYTOUSE   SOURCEPVC   SOURCESNAPSHOTCONTENT   RESTORESIZE   SNAPSHOTCLASS                            SNAPSHOTCONTENT                                    CREATIONTIME   AGE
+rbd-pvc-snapshot   true         rbd-pvc                             1Gi           csi-rbdplugin-snapclass                  snapcontent-905e6015-2403-4302-8a4e-cd3bdf63507b   78s            79s
 ```
 
 ### Check the status of the Snapshot
 
 ```console
 $ kubectl describe volumesnapshot rbd-pvc-snapshot
-
 Name:         rbd-pvc-snapshot
 Namespace:    default
 Labels:       <none>
-Annotations:  <none>
-API Version:  snapshot.storage.k8s.io/v1alpha1
+Annotations:  snapshot.storage.kubernetes.io/pvc-access-modes=ReadWriteOnce
+              snapshot.storage.kubernetes.io/pvc-storage-class=csi-ceph-rbd
```

Comment:
`csi-ceph-rbd` is not the storageclass we create in cephcsi. refer to  https://github.com/ceph/ceph-csi/blob/266e5b3b80a486303cf6bcc48040a4bdc0da8792/examples/rbd/storageclass.yaml#L5?